.github/actions/ci/test/action.yaml

```
@@ -8,23 +8,40 @@ inputs:
     required: true
   codecov_token:
     description: "The Codecov token"
-    required: true
+    required: false
+  no_coverage:
+    description: "disable coverage"
+    required: false
+    default: false
```

Perhaps we leave a comment here about how the coverage doesnt work on windows arm atm -- I know we have a comment for it in the `matrix-prep.py` but I think it's good to have here too (the same comment would probably be fine).
```suggestion
  # currently coverage doesnt work on windows arm
  no_coverage:
    description: "disable coverage"
    required: false
    default: false
```